src/custom-vocabularies-client.ts

```diff
@@ -0,0 +1,57 @@
+import * as FormData from 'form-data';
+import * as fs from 'fs';
+import { Readable } from 'stream';
+
+import { ApiRequestHandler } from './api-request-handler';
+import { CustomVocabulary } from './models/CustomVocabulary';
+import { CustomVocabularyOptions } from './models/CustomVocabularyOptions';
+import { RevAiApiCustomVocabulary } from './models/RevAiApiCustomVocabulary';
+
+/**
+* Client which handles communication with custom vocabularies
+* Rev.ai api
```

Comment:
This doc comment is confusing. The Rev.ai api line seems random and `communication with custom vocabularies` doesn't really mean anything

Reply:
How do you think it is now?